test/Microsoft.AspNetCore.Mvc.Routing.Abstractions.Tests/RouteValueDictionaryTests.cs

```diff
@@ -156,21 +156,6 @@ public void CreateFromObject_CopiesPropertiesFromRegularType_WithIndexerProperty
             Assert.Equal(0, dict.Count);
         }
 
-        [Fact]
-        public void CreateFromObject_MixedCaseThrows()
```

Comment:
Yeah I agree, I think RVD will end up doing it's own cache of PropertyHelper objects. We don't want to lose this error message.